pkg/storage/redis/redis.go

```diff
@@ -352,3 +375,11 @@ func (db *DB) tx(c redis.Conn, commands []map[string][]interface{}) error {
 	_, err := c.Do("EXEC")
 	return err
 }
+
+func (db DB) recordOperation(ctx context.Context, startTime time.Time, operation string, err error) {
```

Comment:
Should this just be a function if db is not being used?

Reply:
Yep.